<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTSMainWindow</class>
 <widget class="QMainWindow" name="DTSMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DTS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="m_pbSysConfig">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="DTS.qrc">
         <normaloff>:/DTS/action_sysconfig</normaloff>:/DTS/action_sysconfig</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="m_pbRunConfig">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="DTS.qrc">
         <normaloff>:/DTS/action_runconfig</normaloff>:/DTS/action_runconfig</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_pbSynConfig">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="DTS.qrc">
         <normaloff>:/DTS/action_synconfig</normaloff>:/DTS/action_synconfig</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="m_pbExit">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="DTS.qrc">
         <normaloff>:/DTS/action_logout</normaloff>:/DTS/action_logout</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QPlainTextEdit" name="m_lteLog">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>780</width>
         <height>400</height>
        </size>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>2000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="m_ActionExit">
   <property name="icon">
    <iconset resource="DTS.qrc">
     <normaloff>:/DTS/action_logout</normaloff>:/DTS/action_logout</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="m_ActionSysConfig">
   <property name="icon">
    <iconset resource="DTS.qrc">
     <normaloff>:/DTS/action_sysconfig</normaloff>:/DTS/action_sysconfig</iconset>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </action>
  <action name="m_ActionRunConfig">
   <property name="icon">
    <iconset resource="DTS.qrc">
     <normaloff>:/DTS/action_runconfig</normaloff>:/DTS/action_runconfig</iconset>
   </property>
   <property name="text">
    <string>运行设置</string>
   </property>
  </action>
  <action name="m_ActionSynConfig">
   <property name="icon">
    <iconset resource="DTS.qrc">
     <normaloff>:/DTS/action_synconfig</normaloff>:/DTS/action_synconfig</iconset>
   </property>
   <property name="text">
    <string>同步设置</string>
   </property>
  </action>
  <action name="m_ActionShow">
   <property name="icon">
    <iconset resource="DTS.qrc">
     <normaloff>:/DTS/main</normaloff>:/DTS/main</iconset>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
   <property name="toolTip">
    <string>显示</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DTS.qrc"/>
 </resources>
 <connections/>
</ui>
